src/System.Runtime/tests/System/Single.cs

```diff
@@ -176,10 +176,6 @@ public static void TestToString()
 
         Single i2 = -8249;
         Assert.Equal("-8249", i2.ToString());
-
-        Assert.Equal("NaN", Single.NaN.ToString());
-        Assert.Equal("Infinity", Single.PositiveInfinity.ToString());
```

Comment:
While I'm happy with the invariant tests, is there an OS API we could call to get the expected value for the general current culture case?